Public/Src/Engine/Scheduler/Artifacts/FileContentManager.cs

```diff
@@ -281,6 +281,10 @@ public bool ReportWorkerPipInputContent(LoggingContext loggingContext, FileArtif
             Contract.Assert(IsDistributedWorker);
 
             SetFileArtifactContentHashResult result = SetFileArtifactContentHash(artifact, info, PipOutputOrigin.NotMaterialized);
+
+            // Notify the host with content that was reported
+            m_host.ReportContent(artifact, info, PipOutputOrigin.NotMaterialized);

```

Comment:
Shouldn't this be ReportDynamicCOntent?

Reply:
It's used down on line 590

Reply:
OOPS :) Sorry missed that change :)

---
In reply to: [286714023](https://github.com/microsoft/BuildXL/pull/361#discussion_r286714023) [](ancestors = 286714023)